src/contexts/playlistContext/playlistReducer.js

```diff
@@ -1,5 +1,26 @@
 const playlistReducer = (playlistState, playlistAction) => {
```

Comment:
this is good to keep reducer in separate file

Reply:
Thanks